test/email_alert_api_test.rb

```diff
@@ -955,52 +955,6 @@
     end
   end
 
-  describe "bulk_migrate" do
-    let(:from_slug) { "i_am_a_subscriber_list_slug" }
-    let(:to_slug) { "i_am_another_subscriber_list_slug" }
-
-    it "returns 202 if valid to_slug and from_slug are provided" do
-      stub_email_alert_api_bulk_migrate(
```

Comment:
Should this method (and similar ones related to the bulk-migrate endpoint) be removed from `lib/gds_api/test_helpers/email_alert_api.rb`?